builtin/moon.pkg.json

```diff
@@ -18,6 +18,8 @@
     "int64_nonjs.mbt": ["not", "js"],
     "arraycore_js.mbt": ["js"],
     "arraycore_nonjs.mbt": ["not", "js"],
-    "array_nonjs_test.mbt": ["not", "js"]
+    "array_nonjs_test.mbt": ["not", "js"],
+    "stringbuilder_buffer.mbt": ["not", ["js"]],
+    "stringbuilder_concat.mbt": ["js"]
```

Comment:
Mark todo: add `wasm-gc` here once we have support for jsstring proposal on wasm-gc